extensions/amp-ima/0.1/validator-amp-ima.protoascii

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Other than the one comment I made it looks fine to me. If a test file is handy, it may be useful to put it in where the validator build expects it.
This is a reasonable example (the files prefixed with "validator-"):
https://github.com/ampproject/amphtml/tree/master/extensions/amp-accordion/0.1/test